### ISSUE TITLE
FMD Launchpad to help launch NFT projects in FTM

### ISSUE THREAD

**Issue by july5613** [FantomMenaceDAO/MIPs#9]
# Menace Improvement Proposal (MIP-9)

## tl;dr
- Build a launchpad program for the DAO where we would support arists/creators projects through devs and marketing to launch in return for whitelisted NFTs and/or % of mint sales
- This would be a good way to provide resources to upcoming NFT artists that they would normally not have access to. 

## Motivation
We should this because it would...
- give upcoming artists a stake in their NFT future
- provide fantom ecosystem exposure to artists that would be not typically receive it
- provide liquidity for DAO and add diversity to treasury at discounted rate. 

### Scope of work / Structure:

**Step 1. Open Application**
- Open for application, rolling acceptance of projects, maybe max 3 at a time
- Setup for artists to apply to the Launchpad Program
- Setup simple website for folks to reach out to (webflow? google sheets)
- Onboarding funnel for artists/creators/devs
- DAO members tweet saying we're open to applications

**Step 2. Application review process**
- Evaluate Application based on following points
- 1) Art evaluation 
- 2) Structure evaluation/ Utilities 
- 3) NFT tokenomics evaluation 
- 4) Difficulty of development on our end 
- 5) Business / Profit evaluation 

**Step 3. Select applicants: Follow up call / Panel interview (if accepted)**
- We are partnering with the projects and artists 
- Put the FMD stamp on them we have to make sure they represent the brand 
- (Optional) Contract - Legal / it’s crypto and if we are developing their site less worried and just for show but good to have something signed by both parties

**From DAO end:** 
- Suggestions on tokenomics, mint size, price 
- Possible beneficial utilities 
- Smart contract development (after first one we will have a template we can use again )
- Website development (same save template for future)
- Suggestion on Discord set up / Twitter set-up 
- Marketing plan 
- Use our social media influence for promotion

**Deal structure:**
- This is how the deal might break down. The applicant may be an artist, dev, PM, etc. in which they would take their place:
 1) 50% - artist (split between whoever participates)
 2) 15% - front end lead 
 3) 15% - smart contracts / generative stuff lead
 4) 10% - marketing / community management
 5) 10% of sales go to the DAO
 6) 5% royalties depending on structure. 
      artist 30%
      marketing 30%
      front end is 10%
      contracts is 10%
      DAO gets 10%

**Step 4. Execution & Launch**
- Turn the application into a separate proposal that internally project owners can self-assign with interest
- Execution of project
- Launch of project / support from DAO

## Potential Contributors / Owners
- @0xJuly 
- @TinfoilRealist
- @DubwiseRenzo
- @FantomCG
- @chronicdispositions
- anyone else who's interested

## Resources / Compensation
Do you need funding? Do the contributors of the project need to be paid in NFTs or FTM? 
- TBD

## Timeline
How long will it this roughly take?
- TBD

## Further Context / support from community members
> @0xJuly: we should look into starting a launchpad program for the DAO where we support community projects launch (provide help, etc.) and in return get whitelisted for mints. after launch the DAO can help promote the projects, support artists, continue to sow the seeds to turn them into potential blue chips if the community moves in that direction. either way, i think it'd be a win for the NFT ecosystem + Fantom + artists/creators
> https://discord.com/channels/893513538633613392/900073013624578048/900073566157041764

> @DubwiseRenzo: Create projects in-house or leverage/fund teams (read DEVS) that we see potential in
> https://discord.com/channels/893513538633613392/900073013624578048/900076282254069801

> MrMike: I think a launchpad format is a great way to go about this
> https://discord.com/channels/893513538633613392/900073013624578048/900921604954001428

> @chronicdispositions: DEA
> aDAOpt an artist
> Up and coming artist doesn't know understand NFT
> DAO grants a contract mentor in exchange for some sort of mint consideration
> Either a piece featuring FMD or X number of pieces given to DAO upon launch
> costs basically nothing
> Gains potentially valuable NFTs and exposure
> https://discord.com/channels/893513538633613392/893518556975755265/903331095070793778

> @FantomCG: It’s a good idea, there’s not ton of solidarity developers out there and not many interested in small projects and developing costs quite high on the front end if you don’t partner with a dev. So I see this being a great source of revenue and ability to easily increase holdings at fraction of costs. And once we have an initial template should be easy to implement on repeat, take a fair % of mint profits and then ask for 5% of mints.
> https://discord.com/channels/893513538633613392/893518556975755265/903345616011296789

**Comment by hotsauce510** [comment]
This sounds like an incubator of sorts and I like it.

**Comment by hashyjim** [comment]
i like the sound of this
